### PR TITLE
Updated issue #235 - Text Alignment for Roles & Package update

### PR DESCRIPTION
**Pull request by Celesica**
#235

### CODE REVIEW

File: lib/features/devboard/devs/devs_list.dart [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import 'package:devs/core/constant/string.dart';
 import 'package:devs/core/models/dev.dart';
+import 'package:flutter/cupertino.dart';
```
Comment (Jansalvador1445) [inline lib/features/devboard/devs/devs_list.dart:3]: I think we don't need this one.

File: pubspec.yaml [hunk 2]
```diff
@@ -9,7 +9,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  cupertino_icons: ^1.0.0
+  cupertino_icons: ^1.0.2
```
Comment (Jansalvador1445) [inline pubspec.yaml:12]: revert to `.0`

**Comment by Jansalvador1445** [comment]
This is good, maybe we need to remove and revert.

File: pubspec.yaml (hunk 2 above)
Comment (Celesica) [reply pubspec.yaml]: reverted~

File: lib/features/devboard/devs/devs_list.dart (hunk 1 above)
Comment (Celesica) [reply lib/features/devboard/devs/devs_list.dart]: thanks!

**Comment by Jansalvador1445** [comment]
This is good, good Job @Celesica ! Wohooo! :D

**Pull request merged**